wiki/Do_you_really_want_to_ask_peppy/en.md

```diff
@@ -1,8 +1,8 @@
 # Do you really want to ask peppy?
 
-Hi there! peppy understands your urge to want to contact him personally, but he asks that you first understand that this is **not** the place you want to ask him for help or support. He keeps osu! support separate from personal messages and also finds that answering forum PMs is quite hard to keep up with. He therefore asks that you do one of the following options:
+Hi there! peppy understands your urge to want to contact him personally, but he asks that you first understand that this is **not** the place you want to ask him for help or support. He keeps osu! support separate from personal messages and also finds that answering forum and PMs are quite hard to keep up with. He therefore asks that you do one of the following options:
 
-## Technical Support
+## Technical Support <!-- i think need to insert: Tech Support in github but i dont know what repo to put in -->
```

Comment:
```suggestion
## Technical Support
```

users should always attempt the help forum first. bugs relating to stable are few and far between, if the support team concurs its a genuine issue then they can direct them to https://github.com/ppy/osu-stable-issues/issues

Reply:
ok i talked this with Civil oath so deleted thx for review